firmware/application/apps/ui_debug.cpp

```diff
@@ -381,10 +381,57 @@ DebugMenuView::DebugMenuView(NavigationView& nav) {
         {"Peripherals", ui::Color::dark_cyan(), &bitmap_icon_peripherals, [&nav]() { nav.push<DebugPeripheralsMenuView>(); }},
         {"Temperature", ui::Color::dark_cyan(), &bitmap_icon_temperature, [&nav]() { nav.push<TemperatureView>(); }},
         {"Buttons Test", ui::Color::dark_cyan(), &bitmap_icon_controls, [&nav]() { nav.push<DebugControlsView>(); }},
+        {"Pmem", ui::Color::dark_cyan(), &bitmap_icon_memory, [&nav]() { nav.push<DebugPmemView>(); }},
     });
     set_max_rows(2);  // allow wider buttons
 }
 
+/* DebugPmemView *********************************************************/
+
+DebugPmemView::DebugPmemView(NavigationView& nav)
+    : data{*reinterpret_cast<pmem_data*>(memory::map::backup_ram.base())}, registers_widget(RegistersWidgetConfig{page_size, 8}, std::bind(&DebugPmemView::registers_widget_feed, this, std::placeholders::_1)) {
+    static_assert(sizeof(pmem_data) == memory::map::backup_ram.size());
+
+    add_children({&text_page, &registers_widget, &text_checksum, &button_ok});
+
+    text_page.set_parent_rect({16, 16, 208, 16});
```

Comment:
Nit: Normally (i.e. in the rest of the code) these dimensions are set in the class header.

Reply:
its done except for registers widget that doesnt have such ctro